lib/game_teams.rb

```diff
@@ -0,0 +1,35 @@
+class GameTeams
```

Comment:
This all looks really good! Good use of data to follow the schema of what we already have.